src/m_menu.c

```diff
@@ -6441,6 +6416,28 @@ void M_StartControlPanel (void)
 
 void M_Drawer (void)
 {
+   static int menushade;
+
+   if (setup_active && menu_background == 2)
+   {
+      int y;
+      byte *dest = screens[0];
+      static int firsttic;
+
+      for (y = 0; y < (SCREENWIDTH << hires) * (SCREENHEIGHT << hires); y++)
+      {
+         dest[y] = colormaps[0][menushade * 256 + dest[y]];
+      }
+
+      if (menushade < 16 && gametic != firsttic)
+      {
+         menushade += 4;
```

Comment:
So, the whole fading takes only 4 tics, isn't this a bit short?

Reply:
I like it when it fading faster. What do you suggest?

Reply:
> I like it when it fading faster. What do you suggest?

I like it when it takes a bit longer. 😛 😉

Reply:
I guess 8 tics is compromise 😄

Reply:
💪